### DISCUSSION TITLE
HILFE: WLAN zu FritzBox bricht ab. Neustart nur nach mehreren Minuten ohne Strom möglich.

### DISCUSSION THREAD

**Discussion by fritzCARaldo** [tbnobody/OpenDTU#1653] [Q&A]
UPDATE 2: Verbindung steht jetzt und Update auf aktuellste Version erfolgt. Jedoch bricht die Verbindung nach einiger Zeit ab. Neustart des openDTU alleine reifht nicht. Ich muss ihn mehrere Minuten vom Strom trennen, dann verbindet er sich wieder ganz normal!? Was ist das? Hat jemand eine Ahnung?

UPDATE: Wollte gerade nochmal flashen und von vorne anfangen und hatte den openDTU dafür mal 10Min. ohne Strom. Gerade als ich loslegen wollte, hat er - wie von Zauberhand - Kontakt aufgenommen... 

Habe mit Tasmota 2.2.3 neu geflashed, aber bekomme den einfach nicht in mein WLAN. Das eigene WLAN wird aufgebaut, ich kann mich verbinden, PW ändern, etc. Aber der übernimmt meine Netzwerkdaten einfach nicht (speichert, baut dann aber wieder das eigene Netz auf).

Ich wollte von v24.1.4 auf OpenDTU Build #1176 updaten. Dieses stoppte bei 19%. Erneuter Versuch bricht bei 9% ab und meldet "OTA-Fehler - [HTTP ERROR] Bad Request". Habe es dann mit OpenDTU Build #1169 versucht, mit dem gleichen Resultat. Nach einem Neustart ist die OpenDTU jetzt gar nicht mehr erreichbar!? Kann mir da jemand helfen? Vielen Dank!

**Comment by NintendoTito** [comment]
OpenDTU mit Tasmota geflasht? Hat dieses Problem auch etwas mit einem "normalen" Update zu tun? Wollte wohl gerne auf die aktuellste Version upgraden.

**Reply by fritzCARaldo** [reply]
Ich habe doch geschrieben, dass das "normale" Update hängengeblieben ist. Habe daraufhin neu geflasht, bekomme das OpenDTU jetzt aber nicht mehr in mein WLAN...

**Comment by fritzCARaldo** [comment]
<img width="562" alt="Bildschirmfoto 2024-01-16 um 18 35 17" src="https://github.com/tbnobody/OpenDTU/assets/118612225/6c0603cb-691b-434a-81f6-4b37b264a30a">

Ich werd kirre... Über das Pop-Up Fenster des  Access Points (OpenDTU-163XXXX) kann ich unter Einstellung->Netzwerk Daten einstellen, aber die funktionieren nicht. Das Fenster schließt sich nach Eingabe und WLAN wechselt, doch dann taucht der Access Points (OpenDTU-163XXXX) wieder auf. Wieder einloggen: Netzwerk INAKTIV. Hab dann mal versucht die Daten über den Browser einzutragen (192.168.4.1.) und da übernimmt er sie. Unter Info-> steht AKTIV! In meiner FritzBox taucht der OpenDTU aber nicht auf und ist auch nicht erreichbar... Das gibt es doch nicht?! Stunden verplempert... Wo ist der (Denk-)Fehler?
